azure-pipelines/release.yml

```diff
@@ -1,16 +1,16 @@
 trigger: none
 pr: none
 
+resources:
```

Comment:
Seems like you are suppose to create release from 'release' branch, instead always from main, is that correct?

Reply:
This is just sets the default branch to pick from, you can actually pick any branch and build you want in the resources screen.

Running this on a main build will upload a prerelease, running on a release branch build will upload a release

Reply:
![image](https://github.com/dotnet/vscode-csharp/assets/5749229/81de78fb-74a8-475e-b506-ca71f9bb12a7)

Reply:
Yeah I am fine with it but just want to check (because now we run everything from main)

Reply:
oh sorry I misunderstood - no the pipeline yaml comes from main still.  Even if the pipeline branch is main you can pick a resource from any branch